libipl/p10/ipl0.C

```diff
@@ -254,13 +254,17 @@ static void update_hwas_state(bool is_coldboot)
 // and of available procs in the system. For few children of master proc
 // functional and present state will be set in device tree during genesis
 // boot.
+// It will also update functional state oscrefclk target.
```

Comment:
This should be fisrt commit and add hwas based state check in the set_ref_clock code .instead of pdbg state check

Reply:
this code need planar callout implemented.

Reply:
Looks like commit ordering is not correct here, enable individaul function first based on dependency ad enable the feature .

Reply:
Done